src/cmds/cloud/publishData.js

```diff
@@ -2,37 +2,28 @@
 import yargs from 'yargs';
 import { Client } from '@cesarbr/knot-cloud-sdk-js';
 import isBase64 from 'is-base64';
+import chalk from 'chalk';
 import options from './utils/options';
 import getFileCredentials from './utils/getFileCredentials';
 
-const publishData = (args) => {
+const publishData = async (args) => {
+  const data = [{ sensorId: args.sensorId, value: args.value }];
   const client = new Client({
-    protocol: args.protocol,
     hostname: args.server,
     port: args.port,
-    pathName: args.pathName,
-    id: args['client-id'],
-    token: args['client-token'],
-  });
-
-  client.on('ready', () => {
-    client.publishData(args['sensor-id'], args.value);
-  });
-  client.on('published', () => {
-    client.close();
-  });
-  client.on('error', (err) => {
-    console.error(err);
-    console.log('Connection refused');
+    token: args.token,
+    protocol: args.protocol,
   });
-  client.connect();
+  await client.connect();
+  await client.publishData(args.thingId, data);
+  await client.close();
 };
 
 yargs
   .config('credentials-file', path => getFileCredentials(path))
   .command({
-    command: 'publish-data <sensor-id> <value>',
-    desc: 'Publish <value> as a <sensor-id>',
+    command: 'publish-data <thing-id> <sensor-id> <value>',
+    desc: 'Publish <sensor-id> <value> as <thing-id>',
```

Comment:
Shouldn't this be the behavior? I mean, we support multiple operations at once and we are limiting the user to a single one at a time... :thinking: 

```suggestion
    command: 'publish-data <thing-id> <data-list>',
    desc: 'Publish <sensor-id> <data-list>',
```

Reply:
I particularly think this limitation turns the usage simpler. If the user wants to send another data, it'll execute the command one more time. Besides, we need to know the sensor-id for each published data.